2. Задача на обробники подій/script.js

```diff
@@ -0,0 +1,68 @@
+  // Отримання посилань 
```

Comment:
```suggestion
'use strict';
  // Отримання посилань 
```

Як я неодноразово наголошував, весь код, завжди, без винятків, має бути написаний в строгому режимі. Так були б помітні помилки зі змінними.